doonut-app-external-api/src/main/kotlin/com/doonutmate/calendar/service/CalendarAppService.kt

```diff
@@ -40,6 +41,19 @@ class CalendarAppService(
         return calendarFacadeService.convertToList(calendarBusinessService.findCalendars(page, time, memberId))
     }
 
+    fun getCalendar(memberId: Long): Calendar {
+        validate(memberId)
+        return calendarBusinessService.getByMemberId(memberId)
+    }
+
+    fun updateCalendar(memberId: Long, title: String) {
+        try {
+            calendarBusinessService.updateCalendarName(memberId, title)
+        } catch (e: CalendarException) {
+            throw CalendarException(ErrorCode.NOT_UPDATED, "캘린더가 업데이트되지 않았습니다.")
+        }
```

Comment:
try-catch문 없애도 될 것 같아요.  

```suggestion
        calendarBusinessService.updateCalendarName(memberId, title)
```